src/ObjCRuntime/Registrar.cs

```diff
@@ -87,9 +88,7 @@ static class Shared {
 		public static ProductException GetMT4127 (TMethod impl, List<TMethod> ifaceMethods)
 		{
 			var msg = new System.Text.StringBuilder ();
-			msg.Append ("Cannot register more than one interface method for the method '");
-			msg.Append (impl.DeclaringType.FullName).Append ('.').Append (impl.Name);
-			msg.Append ("' (which is implementing '");
+			msg.Append (String.Format(Errors.MT4127, impl.DeclaringType.FullName, impl.Name));
 			for (int i = 0; i < ifaceMethods.Count; i++) {
 				if (i > 0)
 					msg.Append (i < ifaceMethods.Count - 1 ? "', '" : "' and '");
```

Comment:
This logic needs to be reworked/refactored a little bit so that we don't end up adding English text (`and`) to translated messages.

Reply:
The full message is: 

`Cannot register more than one interface method for the method '{0}' (which is implementing '{1}'  and '{2}' and '{3}' and ...  '{x}').`

The for loop is creating this part:

`'{1}'  and '{2}' and '{3}' and ...  '{x}'`

Would changing `and`s to commas fix this? 

So the new message would be:

`Cannot register more than one interface method for the method '{0}' (which is implementing: '{1}', '{2}', '{3}', ...  '{x}').`

Reply:
I don't know if that would work with languages that aren't left-to-right (Chinese, Arabic, etc.)

My suggestion would be to emit multiple errors:

> MT4127: Cannot register more than one interface method for the method '{0}'.
> MT41??: The method '{0}' is implementing '{1}'.
> MT41??: The method '{0}' is implementing '{1}'.
> MT41??: The method '{0}' is implementing '{1}'.
> ...

Reply:
I've implemented the multiple errors solution. All of the error codes are `MT4127`, not sure if we want to change `The method '{0}' is implementing '{1}'` to different error codes since they're related.

Reply:
Yes, I think it's best to use a different error code since the text is different.